.circleci/Makefile

```diff
@@ -1,21 +1,20 @@
 # Builder version
-VERSION ?= go1.9-k8s1.7.4
-PROJECT ?= istio
-HUB ?=
+CI_VERSION ?= go1.9-k8s1.7.4
```

Comment:
change this, as you have changed the docker file

Reply:
I think CircleCI will detect the change and pull ( it is a label - like latest ). 
It is only a problem in k8s, where we need to use the sha as a workaround.

If it fails - I'll do that.

Reply:
circleci caches docker images. So if we update the same image, some nodes are going to have latest while others will have old.